From e9303a4eac7260ef204d69b2b3805b438dc2fde7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C4=B0=2E=20Ensar=20G=C3=BCl=C5=9Fen?=
 <ismailensarg@gmail.com>
Date: Sun, 10 Mar 2024 17:52:15 +0300
Subject: [PATCH] hwdb: Add touchpad configuration for ThinkPad E495

(cherry picked from commit d3ca25645fb5ead41c4f924848f105c29d4ecd97)

Related: RHEL-5950
---
 hwdb.d/60-evdev.hwdb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index 4bfea37174..43017dc00f 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -713,8 +713,9 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*:svnLENOVO:*pvrThinkPadP14sGen2a:*
  EVDEV_ABS_35=::44
  EVDEV_ABS_36=::52
 
-# Lenovo ThinkPad T15g Gen1
+# Lenovo ThinkPad T15g Gen1 And E495
 evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO:*pvrThinkPadT15gGen1**
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO:*pvrThinkPadE495**
  EVDEV_ABS_00=::44
  EVDEV_ABS_01=::50
  EVDEV_ABS_35=::44
